app/lib/shared/urls.dart

```diff
@@ -143,6 +143,19 @@ String inferIssueTrackerUrl(String baseUrl) {
   return null;
 }
 
+/// Infer base URL that can be used to link files from.
+String inferBaseUrl({String homepageUrl, String repositoryUrl}) {
```

Comment:
If `repositoryUrl` starts with `https://github.com/` or `https://gitlab.com/` and ends with `.git` then we remove `.git`.

Reply:
Also remove `www` from github and gitlab.